client/src/components/screens/PatientSearch/index.jsx

```diff
@@ -214,12 +214,19 @@ class PatientSearchScreen extends React.Component {
 
     if (searchType !== 'autocomplete') {
       const data = nextProps.search[searchType].results.map((result) => {
+        const organizationValue = () => {
+          if (result.organization.name === '') {
+            const organizationId = result.organization.id.split('/');
+            return organizationId[organizationId.length - 1];
```

Comment:
An organization's id is always `Organization/id`, so you could just return `result.organization.id.split('/')[1]`